### ISSUE TITLE
Error coding

### ISSUE THREAD

**Issue by mahmoud422** [KuChainNetwork/Project-Deluge#6]
On line 110, you notice that this value is not present in the service application In this path https://github.com/KuChainNetwork/kratos/blob/master/third_party/proto/tendermint/abci/types/types.proto

**Comment by cain42** [comment]
Do you mean this line is not present in the service application?

https://github.com/KuChainNetwork/kratos/blob/master/third_party/proto/tendermint/abci/types/types.proto#L110

**Comment by mahmoud422** [comment]
Yes

**Comment by Pisces-Anjou** [comment]
Hi,



Thanks for your submission.

This is a type definition we reserved for expansion, which will not impact the entire program. 

I'm afraid we couldn't get your meaning if you don't describe it in detail. Do you think there will be any bad effects for the program caused by the line you mentioned? Please follow [the vulnerability report template](https://github.com/KuChainNetwork/Project-Deluge/issues/new?assignees=&labels=&template=template.md&title=) and share us more specific information.



Regards

KuChain Team

**Comment by mahmoud422** [comment]
No, there are no major effects, but in the future it may cause a major problem, or soon

**Comment by Pisces-Anjou** [comment]
Hi,

Could you please give some examples to show us in what kind of circumstances will it(the line 110) caused the "major problem" in the future?

Regards
KuChain Team

**Comment by mahmoud422** [comment]
Sorry for being late due to family problems. It may slow servers, and with new updates it may cause servers with cyberattacks to slow network traffic.

**Comment by mahmoud422** [comment]
Sorry for being late due to family problems. It may slow servers, and with new updates it may cause servers with cyberattacks to Drop network traffic.

**Comment by Pisces-Anjou** [comment]
Hi,

Thanks for your replying.
Note that there won't be any practical impact delivered by the issue you mentioned, it will not affect the normal operation of the chain, and it is not in the scope from P1 to P4. After evaluation, we consider it is not a valid vulnerability but a good improve suggestion. 
Thank you very much for your attention. Please keep trying and hep us improve our chain.

Regards
KuChain Team

**Comment by mahmoud422** [comment]
Okay thanks,But I think the P4 vulnerability should be evaluated

**Comment by Pisces-Anjou** [comment]
Hi,

Thanks for your replying.
As what I have said above, there won't be any practical impact delivered by the issue you mentioned, it will not affect the normal operation of the chain, and it is not in the scope from P1 to P4.
What's more, the inferring problems are not in the evaluation scope. What you reported is not a valid bug for KuChain.
Hope you could understand.

Regards
KuChain Team
